components/landing/TheQiskitCapabilitiesSection/index.vue

```diff
@@ -1,16 +1,233 @@
 <template>
-  <article class="bx--grid page-section">
+  <article class="bx--grid page-section qiskit-capabilities-section">
     <h2>What Can Qiskit Do</h2>
-    <CodeCell :active-blocks="activeBlocks" />
+    <div class="qiskit-capabilities-section__container">
+      <div class="qiskit-capabilities-section__capabilities">
+        <div class="qiskit-capabilities-section__content">
+          <QiskitCapabilityCard
+            v-for="item in capabilities"
+            :id="item.title"
+            :key="item.title"
+            class="qiskit-capabilities-section__card scrollable"
+            :title="item.title"
+            :description="item.description"
+            :thumbnail-resource="item.thumbnailResource"
+            :link="item.link"
+          />
+        </div>
+        <div class="qiskit-capabilities-section__scrolling-ui">
+          <CodeCell :active-blocks="capabilities.map((item, index) => isActiveSection(item, index))" />
+        </div>
+      </div>
+    </div>
+    <div class="qiskit-capabilities-section__ctas">
+      <AppCta
+        class="qiskit-capabilities-section__ctas__link"
+        v-bind="quantumLabCTA"
+        kind="ghost"
+      />
+      <cv-button
+        :title="copyCodeCTA.label"
+        class="qiskit-capabilities-section__copy-button"
+        @click="[copyToClipboard(), $trackClickEvent(copyCodeCTA.segment)]"
+      >
+        <span class="qiskit-capabilities-section__copy-button__label">{{ copyCodeCTA.label }}</span>
+        <Copy16 class="qiskit-capabilities-section__copy-button__icon" />
+      </cv-button>
+    </div>
   </article>
 </template>
 
 <script lang="ts">
-import Vue from 'vue'
-import { Component } from 'vue-property-decorator'
+import { Component, Mixins } from 'vue-property-decorator'
+import ScrollSectionsMixin from '~/mixins/scrollBetweenSections'
+
+interface QiskitCapability {
+  /** The visible title of the feature */
```

Comment:
I personally think that this comment is unnecessary since this can be easily inferred from the name of the interface and property.

```suggestion
```